src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -57,12 +59,17 @@ class SonarLintDaemon : ISonarLintDaemon
         private Channel channel;
         private StandaloneSonarLint.StandaloneSonarLintClient daemonClient;
 
-        public SonarLintDaemon() : this(daemonVersion, Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), Path.GetTempPath())
+        
+        [ImportingConstructor]
+        public SonarLintDaemon(ISonarLintSettings settings, ISonarLintOutput logger)
+            : this(settings, logger, daemonVersion, Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), Path.GetTempPath())
```

Comment:
This is a MEF-created component, so it should import any other MEF components it needs, rather than calling _GetMefService_.